environment.yml

```diff
@@ -2,35 +2,32 @@
 # mamba env create -f environment.yml
 name: myria3d
 channels:
-  - conda-forge
-  - anaconda
   - pytorch
-  - comet_ml
+  - nvidia
   - pyg
+  - comet_ml
+  - conda-forge
 dependencies:
-  - python==3.9.*
+  - python=3.9.*
   - pip
-  # --------- data formats --------- #
-  - numpy
-  - h5py
   # --------- Deep Learning --------- #
-  # cudatoolkit to specify the cuda driver in the conda env
```

Comment:
no need of cuda-toolkit anymore?

Reply:
It is replaced with `pytorch-cuda` in later pytorch version